src/features/Applet/Schedule/Calendar/Event/Event.tsx

```diff
@@ -0,0 +1,64 @@
+import { format } from 'date-fns';
+
+import { Svg } from 'components';
+import { DateFormats } from 'consts';
+import { variables } from 'styles/variables';
+import {
+  StyledBodySmall,
+  StyledLabelBoldMedium,
+  StyledLabelMedium,
+} from 'styles/styledComponents/Typography';
+
+import { EventProps, UiType } from './Event.types';
+import {
+  StyledEndIcon,
+  StyledEvent,
+  StyledIndicator,
+  StyledLeftSection,
+  StyledStartIcon,
+  StyledTitle,
+  StyledTopWrapper,
+} from './Event.styles';
+
+export const Event = ({ title, event, uiType = UiType.Primary }: EventProps) => {
+  const { scheduledColor, startFlowIcon, start, end, allDayEvent, endAlertIcon, alwaysAvailable } =
+    event;
+  const isAllDayEvent = allDayEvent || alwaysAvailable;
+
+  return (
+    <StyledEvent>
+      <StyledTopWrapper>
+        <StyledLeftSection>
+          {uiType === UiType.Primary && scheduledColor && (
```

Comment:
would be good move to var uiType === UiType.Primary and uiType === UiType.Secondary

Reply:
done